src/main/java/domain/user/Player.java

```diff
@@ -1,23 +1,50 @@
 package domain.user;
 
-import domain.card.Card;
+import domain.card.Deck;
+import domain.result.GameResult;
 
-import java.util.ArrayList;
-import java.util.List;
+public class Player extends User {
+    private static final boolean STAY = true;
 
-public class Player {
+    private final PlayerName playerName;
+    private final BettingMoney bettingMoney;
+    private boolean isStay = false;
 
-    private final String name;
-    private final double bettingMoney;
-    private final List<Card> cards = new ArrayList<>();
-
-    public Player(String name, double bettingMoney) {
-        this.name = name;
+    public Player(PlayerName playerName, BettingMoney bettingMoney) {
+        this.playerName = playerName;
         this.bettingMoney = bettingMoney;
     }
 
-    public void addCard(Card card) {
-        cards.add(card);
+    public boolean isOwnTurn() {
+        return getStatus().isSurvival() && !isStay;
```

Comment:
```suggestion
        return getStatus() == Status.SURVIVAL && !isStay;
```
위와 같이 변경해볼 수 있을 것 같아요

Reply:
`Status.SURVIVAL`로 `enum`에 정의된 상수를 바로 비교한다면 `getter`로 값을 꺼내어 비교하는 것과 비슷한 형태인 것 같다는 생각이 드는데, `Status` 객체에게 메시지를 보내는 `isSurvival()`로 사용하는게 적합하다고 생각이 들어서 위와 같이 코드를 작성했는데, 바로 비교하는 이유가 있는지 궁금합니다!